.github/workflows/eks-add-on-integ-test.yml

```diff
@@ -83,18 +83,17 @@ jobs:
 
       - name: Confirm EKS Version Support
         run: |
-          if [[ 
+          if [[
             $(go list -m k8s.io/client-go | cut -d ' ' -f 2 | cut -d '.' -f 2) -lt $( echo ${{ matrix.arrays.k8sVersion }} | cut -d '.' -f 2)
             || $(go list -m k8s.io/apimachinery | cut -d ' ' -f 2 | cut -d '.' -f 2) -lt $( echo ${{ matrix.arrays.k8sVersion }} | cut -d '.' -f 2)
             || $(go list -m k8s.io/component-base | cut -d ' ' -f 2 | cut -d '.' -f 2) -lt $( echo ${{ matrix.arrays.k8sVersion }} | cut -d '.' -f 2)
             || $(go list -m k8s.io/kubectl | cut -d ' ' -f 2 | cut -d '.' -f 2) -lt $( echo ${{ matrix.arrays.k8sVersion }} | cut -d '.' -f 2)
-          ]]; then 
+          ]]; then
           echo k8s.io/client-go $(go list -m k8s.io/client-go) is less than ${{ matrix.arrays.k8sVersion }}
           echo or k8s.io/apimachinery $(go list -m k8s.io/apimachinery) is less than ${{ matrix.arrays.k8sVersion }}
           echo or k8s.io/component-base $(go list -m k8s.io/component-base) is less than ${{ matrix.arrays.k8sVersion }}
-          echo or k8s.io/kubectl $(go list -m k8s.io/kubectl) is less than ${{ matrix.arrays.k8sVersion }}, fail test
-          echo "please run go get -u && go mod tidy"
-          exit 1; 
```

Comment:
Cam we add an always() tag to the terraform verification step below so the workflow keeps running and the ```Confirm EKS Version Support``` step result is visible (red/green). I feel just logging the error will just to ignoring the test since it will always show as successful.

```
jobs:
  job1:
  job2:
    needs: job1
  job3:
    if: always()
    needs: [job1, job2]
```
https://stackoverflow.com/a/58859404

Reply:
Hm so I tried adding `always()` to the subsequent steps to allow this step to fail but it then fails the entire workflow: https://github.com/aws/amazon-cloudwatch-agent-operator/actions/runs/10357729837